### PR TITLE
Add Docker image for running bot notebook

### LINKED ISSUE #3: Create a conda or docker environment for the bot runner
pandas, slack, widgets, dashboard, notebook, oh my!

### PR DESCRIPTION
**Pull request by jtyberg**
Also includes conda environment file to enable running admin script.

Fixes #3.

### CODE REVIEW

File: admin/environment.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: pydata
+dependencies:
+- openssl=1.0.2h=1
+- pip=8.1.2=py35_0
+- python=3.5.2=0
```
Comment (parente) [inline admin/environment.yml:3]: Probably should specify only the python version. The rest gets installed automatically I think.

**Comment by jtyberg** [comment]
Added lite version of environment.yml.

**Comment by parente** [comment]
Fine by me. I have no way to test it, so merge when you're happy.

**Comment by jtyberg** [comment]
@lbustelo is testing it.  He hit a snag.  Will merge when he's done.

**Comment by lbustelo** [comment]
Tested the bot notebook and was able to connect to SL no errors.

LGTM!

**Pull request merged**